### PR TITLE
Make getCommitHash  testable 

### PR DESCRIPTION
**Pull request by chaspy**
ref: https://github.com/kamontia/qs/issues/115

### CODE REVIEW

**Comment by chaspy** [comment]
@kamontia Please review and merge

File: model/data.go [hunk 1]
```diff
@@ -60,3 +61,19 @@ func (g *GitCommitInfo) AddCommitMessage(msg string) {
 		g.CommitSpecifiedMsgList = append(g.CommitSpecifiedMsgList, fmt.Sprintf("fixup! %s", msg))
 	}
 }
+
+// AddCommitHash insert commit hash
+func (g *GitCommitInfo) AddCommitHash() int {
```
Comment (kamontia) [inline model/data.go]: It seems not to be important to store commit hash in the main process.
I think that it is divided into  two functions following as : 

-  return `headMax`
-  store commit hash when called by `qs ls` only

Could you improve this function?

Comment (chaspy) [reply model/data.go]: addressed.

>store commit hash when called by qs ls only

AddCommitHash is called from check current commit,
so It is always called when qs is executed.

Comment (kamontia) [reply model/data.go]: It seems to be fine. Thank you.

**Comment by kamontia** [comment]
@chaspy Thank you for quick work.  Let's merge!!!

**Pull request merged**